### ISSUE TITLE
Windows 2012/2016 'core' edition indistinguishable from 'gui' editions 

### ISSUE THREAD

**Issue by spuder** [chef/ohai#1093]
ohai has no way of differentating  'headless' (aka 'windows core edition') and 'gui based' windows installations. 

Here is from a windows 2016 Standard Core server

'Core' shows up nowhere in ohai
```
PS C:\Users\foo> ohai | select-string 'Core'
[2017-11-20T23:25:31-05:00] INFO: The plugin path C:\chef\ohai\plugins does not exist. Skipping...

      "cores": 1,
      "cores": 1,
      "cores": 1,
      "cores": 1,
    "cores": 4,
```

The OS Name is the same regardless if 'Standard' or 'Standard Core' 
```
PS C:\Users\foo> ohai | select-string '2016'
[2017-11-20T23:25:43-05:00] INFO: The plugin path C:\chef\ohai\plugins does not exist. Skipping...

      "caption": "Microsoft Windows Server 2016 Standard",
      "name": "Microsoft Windows Server 2016 Standard|C:\\Windows|\\Device\\Harddisk0\\Partition2",
    "name": "Microsoft Windows Server 2016 Standard",
```

Some windows features wont install on 'core' edition. There needs to be a way to make a cookbook work on both 'gui' and 'core' windows. 

The current work around
```ruby
windows_feature 'IIS-ManagementConsole' do
  action :install
  only_if 'Test-Path $env:windir\explorer.exe'
end
```

To detect if the server is running core

https://serverfault.com/questions/529124/identify-windows-2012-server-core

**Comment by tas50** [comment]
Fixed by #1142

**Comment by tas50** [comment]
This was merged to master and will ship in Ohai/Chef 14
